docs/api-usage.md

```diff
@@ -0,0 +1,158 @@
+    - [Basic usage](#basic-usage)
+    - [Activity creation options](#activity-creation-options)
+    - [Adding Events](#adding-events)
+    - [Setting Status](#setting-status)
+  - [Instrumenting a library/application with OpenTelemetry.API Shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim)
+
+## Introduction to OpenTelemetry .NET API
+
+.NET runtime had `Activity` class for a long time, which was meant to be used for tracing purposes and represents the equivalent of the OpenTelemetry [Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span). OpenTelemetry .NET is reusing the existing `Activity` and associated classes to represent the OpenTelemetry `Span`. This means, users can instrument their applications/libraries to emit OpenTelemetry compatible traces by using just the .NET Runtime.
+
+The `Activity` and associated classes are shipped as part of `System.Diagnostics.DiagnosticSource` nuget package. Version 5.0.0 of this package contains improvements to `Activity` class which makes it more closely aligned with OpenTelemetry [API specification](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md).
+
+Even though `Activity` enables all the scenarios OpenTelemetry supports, users who are already familiar with OpenTelemetry terminology may find it easy to operate with that terminology. For instance, `StartSpan` may be preferred over `StartActivity`. To help with this transition, the OpenTelemetry.API package has [shim](#instrumenting-a-libraryapplication-with-opentelemetryapi-shim) classes to wrap around the .NET `Activity` classes.
+
+The shim exist only in the API. OpenTelemetry SDK for .NET will be operating entirely with `Activity` only. Irrespective of whether shim classes or `Activity` is used for instrumentation, the end result would be same. i.e Processors/Exporters see the same data.
+
+## Instrumenting a library/application with .NET Activity API
+
+### Basic usage
+
+As mentioned in the introduction, the instrumentation API for OpenTelemetry .NET is the .NET `Activity` API. Guidance for instrumenting using this API is documented fully in the TBD(dotnet activity user guide link), but is described here as well.
+
+1. Install the `System.Diagnostics.DiagnosticSource` package version 5.0.0-preview.7.20308.13 or above to your application or library.
+
+```xml
+    <ItemGroup>
+        <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="5.0.0-preview.7.20308.13" />
+    </ItemGroup>
+```
+
+2. Create an `ActivitySource`, providing the name and version of the library/application being instrumented. `ActivitySource` instance is typically created once and is reused throughout the application/library.
+
+```csharp
+    static ActivitySource activitySource = new ActivitySource("companyname.product.library", "semver1.0");
+```
+    The above requires import of the `using System.Diagnostics;` namespace.
```

Comment:
Should we just call `System.Diagnostics`  instead of `using System.Diagnostics;`?